### PR TITLE
Update rootcacertificates-csp.md

### PR DESCRIPTION
**Pull request by ojrb**
This PR Solved the Issue #3290 I clarify each principal Node, please review and merge

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/client-management/mdm/rootcacertificates-csp.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ The RootCATrustedCertificates configuration service provider enables the enterpr
  
 The following image shows the RootCATrustedCertificates configuration service provider in tree format.
 
+Here the detailed specfiication of the principal root nodes:
```
Comment (illfated) [inline windows/client-management/mdm/rootcacertificates-csp.md]:
```suggestion
Detailed specification of the principal root nodes:
```

File: windows/client-management/mdm/rootcacertificates-csp.md [hunk 2]
```diff
@@ -49,39 +50,27 @@ Node for trusted people certificates.
 Addeded in Windows 10, version 1803. Node for certificates that are not trusted. IT admin can use this node to immediately flag certificates that have been compromised and no longer usable.
 
 <a href="" id="certhash"></a>**_CertHash_**  
-Defines the SHA1 hash for the certificate. The 20-byte value of the SHA1 certificate hash is specified as a hexadecimal string value.
+Defines the SHA1 hash for the certificate. The 20-byte value of the SHA1 certificate hash is specified as a hexadecimal string value. This node is common for all the principal root nodes.  The supported operations are Get and Delete.
 
-The supported operations are Get and Delete.
+The following nodes, are all common to the **_CertHash_** node:
```
Comment (illfated) [inline windows/client-management/mdm/rootcacertificates-csp.md]:
```suggestion
The following nodes are all common to the **_CertHash_** node:
```

**Comment by illfated** [comment]
@Tdiwestyguy: May we have your comments on this PR content and if this is the sort of change you were asking for in your issue ticket #3290 ?

File: windows/client-management/mdm/rootcacertificates-csp.md [hunk 3]
```diff
@@ -21,6 +21,8 @@ The RootCATrustedCertificates configuration service provider enables the enterpr
  
 The following image shows the RootCATrustedCertificates configuration service provider in tree format.
 
+Here the detailed specfiication of the principal root nodes:
+
 ![roocacertificate](images/provisioning-csp-rootcacertificate.png)
```
Comment (illfated) [inline windows/client-management/mdm/rootcacertificates-csp.md]: @JohanFreelancer9 : What do you think of this part ([roocacertificate])? Even if it is slightly outside of the PR content, should we try to find out if this needs editing?
```
![roocacertificate](images/provisioning-csp-rootcacertificate.png)
```
( roocacertificate -> rootcacertificate )

Comment (JohanFreelancer9) [reply windows/client-management/mdm/rootcacertificates-csp.md]: Agree - roocacertificate -> rootcacertificate.

File: windows/client-management/mdm/rootcacertificates-csp.md [hunk 4]
```diff
@@ -21,6 +21,8 @@ The RootCATrustedCertificates configuration service provider enables the enterpr
  
 The following image shows the RootCATrustedCertificates configuration service provider in tree format.
 
+Detailed specification of the principal root nodes:
+
 ![roocacertificate](images/provisioning-csp-rootcacertificate.png)
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/rootcacertificates-csp.md]:
```suggestion
![rootcacertificate](images/provisioning-csp-rootcacertificate.png)
```

**Comment by illfated** [comment]
@Tdiwestyguy: Are you available for comments on this PR content?
Does this PR contain the changes you asked for?

**Comment by e0i** [comment]
@manikadhiman The PR that addressess issue #3290 is available for final review. Thanks.

**Comment by ManikaDhiman** [comment]
Hi @e0i, From #3290, the doc should be updated to depict nodes with their complete paths and their respective descriptions. 

For example:
**RootCATrustedCertificates/Root**
Defines the certificate store that contains root, or self-signed certificates, in this case, the computer store.

**RootCATrustedCertificates/Root/_CertHash_**
Defines the SHA1 hash for the certificate. The 20-byte value of the SHA1 certificate hash is specified as a hexadecimal string value.
The supported operations are Get and Delete.

**RootCATrustedCertificates/Root/_CertHash_/EncodedCertificate**
Specifies the X.509 certificate as a Base64-encoded string. The Base-64 string value cannot include extra formatting characters such as embedded linefeeds, etc.
The supported operations are Add, Get, and Replace.

**RootCATrustedCertificates/Root/_CertHash_/IssuedBy**
Returns the name of the certificate issuer. This is equivalent to the Issuer member in the CERT_INFO data structure.
The only supported operation is Get.

**RootCATrustedCertificates/Root/_CertHash_/IssuedTo**
Returns the name of the certificate subject. This is equivalent to the Subject member in the CERT_INFO data structure.
The only supported operation is Get.

**RootCATrustedCertificates/Root/_CertHash_/ValidFrom**
Returns the starting date of the certificate's validity. This is equivalent to the NotBefore member in the CERT_INFO data structure.
The only supported operation is Get.

**RootCATrustedCertificates/Root/_CertHash_/ValidTo**
Returns the expiration date of the certificate. This is equivalent to the NotAfter member in the CERT_INFO data structure.
The only supported operation is Get.

**RootCATrustedCertificates/Root/_CertHash_/TemplateName**
Returns the certificate template name.
The only supported operation is Get.

You can follow the same model for CA, TrustedPublisher, TrustedPeople, and UntrustedCertificates nodes.
I understand this will involve replicating the same details under each parent node, but this will bring consistency and completeness to the doc.

@Tdiwestyguy: Please confirm if the above mentioned changes are the desired updates per #3290.
Thanks!

**Comment by Tdiwestyguy** [comment]
Thanks!

Looks good.



________________________________

**Pull request merged**